packages/big-design/src/components/Worksheet/Cell/Cell.tsx

```diff
@@ -158,18 +164,19 @@ const InternalCell = <T extends WorksheetItem>({
         );
     }
   }, [
+    type,
     cell,
-    disabled,
-    formatting,
+    isEditing,
     handleBlur,
     handleChange,
-    handleKeyDown,
-    isEdited,
-    isEditing,
     options,
+    formatting,
     rowId,
+    disabled,
+    editWithValue,
+    isEdited,
+    handleKeyDown,
     renderedValue,
```

Comment:
We prefer to `abc`.

Reply:
@jorgemoya that was the linter auto-fix, I have returned it back))